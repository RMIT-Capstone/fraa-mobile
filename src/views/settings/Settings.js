import React from 'react';
import {View, Text} from 'react-native';
import styles from './SettingsStyle';

const Settings = () => {
  return (
    <View style={styles.centered}>
      <Text>This is settings</Text>
    </View>
  );
};

export default Settings;
