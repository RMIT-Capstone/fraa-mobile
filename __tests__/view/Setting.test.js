// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import Settings from '../../src/views/settings/Settings';
it('renders Settings correctly', () => {
  const tree = renderer.create(<Settings />).toJSON();
  expect(tree).toMatchSnapshot();
});
