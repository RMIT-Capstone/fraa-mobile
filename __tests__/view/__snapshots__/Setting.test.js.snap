// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Settings correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    This is settings
  </Text>
</View>
`;
